rust-toolchain

```diff
@@ -0,0 +1 @@
+1.42.0
```

Comment:
This is caused by a [cargo #8258](https://github.com/rust-lang/cargo/issues/8258)